pkgs/development/python-modules/tidalapi/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+, python-dateutil
+, requests
+, ffmpeg-python
+, keyring
+, pillow
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "tidalapi";
+  version = "0.7.0";
```

```suggestion
  version = "0.7.0";
  format = "setuptools";
```

Set the format explicitly, we're going to require this in the future.